src/Offer/TypeResolverInterface.php

```diff
@@ -7,9 +7,5 @@
 
 interface TypeResolverInterface
 {
-    /**
-     * @param StringLiteral $typeId
-     * @return EventType
-     */
-    public function byId(StringLiteral $typeId);
+    public function byId(StringLiteral $typeId): EventType;
```

Comment:
I know it's weird that the interface returns `EventType` and not some generic type, but originally we only had `EventType` and `Place` was only added later. Apparently this was never renamed to something more generic and its pretty deeply ingrained now so not worth renaming it for this particular (small) change.